core/src/test/scala/org/apache/spark/sql/delta/DeltaSuite.scala

```diff
@@ -1003,6 +1003,62 @@ class DeltaSuite extends QueryTest
     }
   }
 
+  test("batch write: append, dynamic partition overwrite + CDF") {
```

Comment:
can this test be moved to one of the CDC related test suites? i have a feeling that this DeltaSuite is becoming unfiltered dumping ground for tests